scripts/test/support.py

```diff
@@ -157,7 +158,9 @@ def run_command(cmd, expected_status=0, stderr=None, expected_err=None):
   out, err = proc.communicate()
   if proc.returncode != expected_status:
     raise Exception(('run_command failed', err))
-  if expected_err is not None and err != expected_err:
+  err_correct = expected_err is None or \
```

Comment:
Oh, I see. Yeah, that's hard to remove. Looks good then, maybe add a comment though to say why we just check for the subset (if there isn't one already).